lncrawl/binders/epub.py

```diff
@@ -116,19 +122,34 @@ def make_chapter_images(book, image_output_path):
     # end if
 
     for filename in os.listdir(image_output_path):
-        if not filename.endswith('.jpg'):
+        if not (filename.endswith('.jpg') or filename.endswith('.png')):
```

Comment:
it can never contain png image. this check is entirely unnecessary. check here for how the files are being saved: https://github.com/dipu-bd/lightnovel-crawler/blob/master/lncrawl/core/downloader.py#L159